ci/format.sh

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env bash
+
```

Comment:
This file is modified from https://github.com/ray-project/ray/blob/master/ci/lint/format.sh, can we skip add header?

Reply:
OK. Although, I suppose we don't need most of code in this script, but simply run formatting per language implementation, which should be some oneliners.